examples/wearable/UIComponents/contents/navigationelements/pageindicator/index.html

```diff
@@ -20,19 +20,19 @@
 	<div class="ui-page" data-go-to-top-button="true">
 		<header class="ui-header">
 			<h2 class="ui-title">
-				Indicator
+				Page indicator
 			</h2>
 		</header>
 		<div class="ui-content">
 			<ul class="ui-listview">
 				<li>
-					<a href="pageindicator.html">
-						Linear
+					<a href="pageindicator_circle.html">
```

Comment:
How about renaming pageindicator_circle.html to pageindicator-circle.html?

Reply:
done